src/plugins/audioplayers/player.c

```diff
@@ -0,0 +1,413 @@
+#include "gst/gst.h"
+#include "gst/gstelementfactory.h"
+#include "gst/gstmessage.h"
+#include "gst/gstsegment.h"
+#include "platformchannel.h"
+#include <flutter-pi.h>
+#include <plugins/audioplayers.h>
+#include <stdio.h>
+
+FILE_DESCR("AudioPlayer::player")
```

Comment:
```suggestion
FILE_DESCR("audioplayers plugin")
```

That should just be something that's easily recognizable in the cmdline output